modules/l4proxyprotocol/handler.go

```diff
@@ -54,10 +55,12 @@ func (Handler) CaddyModule() caddy.ModuleInfo {
 
 // Provision sets up the module.
 func (h *Handler) Provision(ctx caddy.Context) error {
-	for _, s := range h.Allow {
-		_, n, err := net.ParseCIDR(s)
+	repl := caddy.NewReplacer()
+	for _, allowCIDR := range h.Allow {
+		rs := repl.ReplaceAll(allowCIDR, "")
```

Comment:
```suggestion
		allowCIDR = repl.ReplaceAll(allowCIDR, "")
```
(Again, just wondering if we need a new var? These are nit-picks, but I do find more descriptive names and less variables to be easier to read and follow.)